Sources/HipMobileUI.iOS/HipMobileUI.iOS.csproj

```diff
@@ -186,16 +186,45 @@
     <ITunesArtwork Include="iTunesArtwork" />
     <ITunesArtwork Include="iTunesArtwork@2x" />
     <None Include="packages.config" />
-    <BundleResource Include="Resources\fonts\AlteSchwabacher.ttf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </BundleResource>
     <BundleResource Include="Resources\ic_done.png" />
     <BundleResource Include="Resources\ic_done_2x.png" />
     <BundleResource Include="Resources\ic_done_3x.png" />
-    <BundleResource Include="Resources\Contents.json" />
```

Comment:
These 3 resources should not be removed. Sayanti added them for the additional information button.